samples/sample_common/include/sample_utils.h

```diff
@@ -47,7 +47,8 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 #include "avc_nal_spl.h"
 
 #ifdef ENABLE_MCTF
-#include  <stdexcept>
+#include <stdexcept>
```

Comment:
how stdexcept and mutex include files depend on MCTF feature? I suggest to avoid overuse of ifdefs whenever possible, this makes code simpler. I don't see harm to always have these headers.

Reply:
I think idea was to drop headers used only by MCTF (MCTF code is also under define in this file). Agree with you to drop extra preprocessor logic

Reply:
Done